zipkin-lens/src/components/DependenciesPage/ExplainBox.jsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    color: theme.palette.text.secondary,
+  },
+  iconWrapper: {
+    fontSize: '10rem',
+    lineHeight: '1',
+  },
+  description: {
+    marginTop: theme.spacing(1.4),
+    textAlign: 'center',
+  },
+}));
+
+const ExplainBox = React.memo(() => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
+      <Box className={classes.iconWrapper}>
+        <Box className="fas fa-search" />
+      </Box>
+      <Typography variant="h4">
+        Search Dependencies
+      </Typography>
+      <Typography variant="body1" className={classes.description}>
+        Please select the start time and the end time,
```

Comment:
less words same meaning. Notice I end the sentence because using 'and' in different ways can be tricky. I instead later use 'Then,'

https://guidetogrammar.org/grammar/conjunctions.htm

```suggestion
        Please select the start and end time.
```